CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.1.0
+
+## Package Updates
```

Comment:
`Package Updates` would normally refer to version changes in the package. I'd use `Improvements` as a heading instead.